### ISSUE TITLE
Flyway 7.10.0 failing in docker brew

### ISSUE THREAD

**Issue by julopezc-acciona** [Homebrew/homebrew-core#79559]
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```shell
# brew gist-logs flyway
Error: No logs.

# brew config
HOMEBREW_VERSION: 3.1.12-87-g04e09de
ORIGIN: https://github.com/Homebrew/brew
HEAD: 04e09de1d84906d97230fe305a492aa802f4f9d9
Last commit: 3 hours ago
Core tap ORIGIN: https://github.com/Homebrew/linuxbrew-core
Core tap HEAD: 35fe9f8543ab7e398ee038e3dda431c49de8d37c
Core tap last commit: 12 hours ago
Core tap branch: master
HOMEBREW_PREFIX: /home/linuxbrew/.linuxbrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.3 => /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.3_2/bin/ruby
CPU: octa-core 64-bit skylake
Clang: N/A
Git: 2.32.0 => /bin/git
Curl: 7.68.0 => /usr/bin/curl
Kernel: Linux 5.11.0-18-generic x86_64 GNU/Linux
OS: Ubuntu 20.04.2 LTS (focal)
Host glibc: 2.31
/usr/bin/gcc: 9.3.0
/usr/bin/ruby: N/A
glibc: N/A
gcc@5: N/A
xorg: N/A


# brew doctor
Your system is ready to brew.
```

### 

- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.

### What were you trying to do (and why)?

Install flyway

### What happened (include all command output)?

```

# brew install flyway
==> Downloading https://ghcr.io/v2/linuxbrew/core/alsa-lib/manifests/1.2.5-1
Already downloaded: /root/.cache/Homebrew/downloads/2097ba3e215c543d5b992967228d2ea6958776c602e339e1a89214260f064468--alsa-lib-1.2.5-1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/alsa-lib/blobs/sha256:b91e5524748677d744b486196e343f46916e76fee9ca945591809f32197cf45d
Already downloaded: /root/.cache/Homebrew/downloads/7e7d0e6822aff49f3d866de7bb346d1e582e1ac129978815d4968704b76d6ffb--alsa-lib--1.2.5.x86_64_linux.bottle.1.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/openssl/1.1/manifests/1.1.1k
Already downloaded: /root/.cache/Homebrew/downloads/c91f32e2a38e6433165dcf2bf0e14632986831fb724916e4db3e86c941d53e2f--openssl@1.1-1.1.1k.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/openssl/1.1/blobs/sha256:d7b7cc5b85d3c73a07e17a25d21f5bf805112dc8df298664e51b8722dff0a667
Already downloaded: /root/.cache/Homebrew/downloads/8ce87933754ec49bbf96f9b36047ca8875b2dfc09c6e67b1f87ee40fb5fd64fb--openssl@1.1--1.1.1k.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/m4/manifests/1.4.18-1
Already downloaded: /root/.cache/Homebrew/downloads/a1363dc62cbd525f9b67afea27c58117af298f6b3af282cb4555a5e494ded021--m4-1.4.18-1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/m4/blobs/sha256:1552dd0379252680e0c32085df6b6ab59aa32f45be389ff18620a073a8f3c78c
Already downloaded: /root/.cache/Homebrew/downloads/60c87c44d8737322d0150352d30fdec839218dcb3c0afc605cf91f9067f44d03--m4--1.4.18.x86_64_linux.bottle.1.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/bison/manifests/3.7.6-1
Already downloaded: /root/.cache/Homebrew/downloads/1b9b0a382cee0e5da494afa758bf6611d5bf8d9613fa6cbd09e3479a17f03a15--bison-3.7.6-1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/bison/blobs/sha256:223e8a0d0ccda242cb92668da34874e273f3f51da657d4b40f449b1e7cbd6e64
Already downloaded: /root/.cache/Homebrew/downloads/a04f059f2880bc0d9350731fc6e7b2e67a6ed107c3561b93b55b9dba15108d99--bison--3.7.6.x86_64_linux.bottle.1.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/ncurses/manifests/6.2
Already downloaded: /root/.cache/Homebrew/downloads/4bb568412133ec15e8798079b09692c9a23dec1fc4e143b43c614c53abc5613d--ncurses-6.2.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/ncurses/blobs/sha256:fc5a0983a8afae8ef46945f3bec9dba9d66e81e879027d100d05088ea6f8043d
Already downloaded: /root/.cache/Homebrew/downloads/8e41e1f16c5dc2d289527bbe8ea35a9914c88b8f0f16d2e4803577e09990990e--ncurses--6.2.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/readline/manifests/8.1
Already downloaded: /root/.cache/Homebrew/downloads/9e8c668c88e063957f830c9b90d8102e00c69690373f708621128a6931a7111c--readline-8.1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/readline/blobs/sha256:cf7ae93f4dc5560c7c7fac35d3fbb7f528a43f3084daf2273e67032d131bdb04
Already downloaded: /root/.cache/Homebrew/downloads/3537e3a851c81a193a47ab1420b33c0c0857f8491f8ebacaead3ce02aca34eec--readline--8.1.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/zlib/manifests/1.2.11
Already downloaded: /root/.cache/Homebrew/downloads/dc8fb21651d58fed8fc68a388299916f3b34a1e46c10c8ca7ff4998117426103--zlib-1.2.11.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/zlib/blobs/sha256:6f604280b9e056da163b81cef3076b2e39bbd2b8d3cc8c36d27c19c472c760a4
Already downloaded: /root/.cache/Homebrew/downloads/79b8196af7ee4821cf6ffdea650eaad7b488c84076fd8259a4ff236c871e82df--zlib--1.2.11.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/libxml2/manifests/2.9.12
Already downloaded: /root/.cache/Homebrew/downloads/2b43dff34aaeb179aaea4ed3c36f0733ab983a69b4034caebbd791ddf594e8f0--libxml2-2.9.12.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/libxml2/blobs/sha256:2b4091b5697e5011a8bd1ef89e762e73a8102dfdfd99632fbc5f1a4182c607bb
Already downloaded: /root/.cache/Homebrew/downloads/6ea61807544f65c008ad315f2da478e47211407f828bd99341fb3e6b01147e40--libxml2--2.9.12.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/gettext/manifests/0.21_2
Already downloaded: /root/.cache/Homebrew/downloads/250eac4f91d85e7c98c4248410cfea3fecc686bf61d948452d8d6ddc4f5ae043--gettext-0.21_2.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/gettext/blobs/sha256:92b2afc5a81a3e51d8da1bdd528af989b60d05a7cf0944479199db89d37adfd7
Already downloaded: /root/.cache/Homebrew/downloads/7620940dfa39e33404d16c6e7ede00d9273fbcdafa717197fc235dd73157ce9c--gettext--0.21_2.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/krb5/manifests/1.19.1_2
Already downloaded: /root/.cache/Homebrew/downloads/300cc346cf2ad72199c98315ed145e30df790e239f039fb025563bc3bef510e1--krb5-1.19.1_2.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/krb5/blobs/sha256:f7ed21e723fe9053cd2c343ac8c225b04da6325e83e9c98758b5ea0d5148b0fe
Already downloaded: /root/.cache/Homebrew/downloads/aa9ddf3a5bd41246ca89d21b3631121552fb3166dcfd540b29fe9c3e438d04fc--krb5--1.19.1_2.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/cups/manifests/2.3.3op2
Already downloaded: /root/.cache/Homebrew/downloads/c3468667ddb529803a5403b5b542cd1918a02360ea5321601b5dcf6ab33fa358--cups-2.3.3op2.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/cups/blobs/sha256:c4ca222a2de10b8a371de6576a7c11de8f0d81b5ddb2bcac76df3e9431b2637c
Already downloaded: /root/.cache/Homebrew/downloads/7e31e04c471b458bc4e0c4431f904ea71f85b643efaf083c3e6e438321a8b55c--cups--2.3.3op2.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/libpng/manifests/1.6.37
Already downloaded: /root/.cache/Homebrew/downloads/9b67cee9aa8bdc2c262c1e0addd5b9e89add61ef39f609b0324300619ad662b7--libpng-1.6.37.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/libpng/blobs/sha256:c5d219b27780c730af50693110ed2c79e50200d3595c7e395998fe3950354ec9
Already downloaded: /root/.cache/Homebrew/downloads/d6033b476784eb5197d663eec0dde087727b4492280a2bc6c31bb09ab4e913e6--libpng--1.6.37.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/bzip2/manifests/1.0.8-1
Already downloaded: /root/.cache/Homebrew/downloads/47621f6c42f43a2cd6372bbf3543c1c5173f32bcb1d02a39826a0d99a432ad8b--bzip2-1.0.8-1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/bzip2/blobs/sha256:2a4cbb681a3aeda2361625bd10d329e47bf986feaccd50bf3a1dce7753ad42d6
Already downloaded: /root/.cache/Homebrew/downloads/078377a3553fa6d93748cd2888dfff050d2e046d434856a3c7950bcec03cee2c--bzip2--1.0.8.x86_64_linux.bottle.1.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/freetype/manifests/2.10.4
Already downloaded: /root/.cache/Homebrew/downloads/068ccea5e4145e5c9d80c2c34c17de5f306aad7be302cc9680e77c6c321afbd5--freetype-2.10.4.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/freetype/blobs/sha256:4b510049ab0cac10c3cd21c481ae2275d506de200161649903a7eff3241b1b26
Already downloaded: /root/.cache/Homebrew/downloads/8bcf78d87ecf4db10049f6a4e8791fcb834bd4e72cd65cd02b4807826469b363--freetype--2.10.4.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/expat/manifests/2.4.1
Already downloaded: /root/.cache/Homebrew/downloads/0b59f69ee922ceb3c10b0ceb329c8671addd426c76e082dd1d625c553da8584f--expat-2.4.1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/expat/blobs/sha256:6cb9d218fe6fbfb86d5aa69f6ac7087b6a082e9b6739a51a701c1a7f3d7d3642
Already downloaded: /root/.cache/Homebrew/downloads/808fdbf356a375b82bb53db432a265895422a72d8f3227077b0c91374a8e8fe3--expat--2.4.1.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/util-linux/manifests/2.37
Already downloaded: /root/.cache/Homebrew/downloads/719bc850a22e8ee11c4acd7a64e3f1e2142e9530887f42fc3e7773cef0242967--util-linux-2.37.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/util-linux/blobs/sha256:24bd0096b100a3734db440f45689524f7f9dbb2896ae77cf2089e91269d11307
Already downloaded: /root/.cache/Homebrew/downloads/f1577f70e1f544a10e098771a3c28f44fb4df1f7c92b10a23f4be7ecff6faa4c--util-linux--2.37.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/fontconfig/manifests/2.13.1_1
Already downloaded: /root/.cache/Homebrew/downloads/1aa40b11cbe3f7a35f52c064f84386d9f411fb1f8a02c6d3db7d6e1131a45e44--fontconfig-2.13.1_1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/fontconfig/blobs/sha256:2aa79674e60e88f1afc613ed65d8e6e0e32f9b19eee7df7afb394b6357555caf
Already downloaded: /root/.cache/Homebrew/downloads/e758603968b6ec7bcf90cb6c86ac1282ecc3f61c09a3a670545874cbf2344dfa--fontconfig--2.13.1_1.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/gmp/manifests/6.2.1
Already downloaded: /root/.cache/Homebrew/downloads/97ccd40c7d7a1aa3ffffd2d54e8bf01b1d57b566bc8686377cf771106a3afefb--gmp-6.2.1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/gmp/blobs/sha256:47ec066627ce0564e7c6b5da2a3311cdcbf09762dce6bad3f0cad46c1377aabc
Already downloaded: /root/.cache/Homebrew/downloads/893618b4697e7877c25243e434ba3f6cf542887026448992d9f49be2c0504313--gmp--6.2.1.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/isl/manifests/0.24
Already downloaded: /root/.cache/Homebrew/downloads/23f9b02c1570fa86f7c84f2a173f21ca57f1785bf1e017b1ee0b51f3818f8999--isl-0.24.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/isl/blobs/sha256:7b58d25de0b7ed797118f32febe116ee2d7458767e6339d6a4ce51ff4ab11191
Already downloaded: /root/.cache/Homebrew/downloads/ae8947b49e5ea6ddcd0621b3993bfddd38cf4e65bdf9997af3017d317fa7e145--isl--0.24.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/mpfr/manifests/4.1.0
Already downloaded: /root/.cache/Homebrew/downloads/4181e9eec7dc6289745c4a1f1d77b7cfea1870bd902668d132410912d7e583ff--mpfr-4.1.0.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/mpfr/blobs/sha256:5fa04c762dd682c46b3c15d2c28e89a36353717c533e7aaa12298b5202aeeba8
Already downloaded: /root/.cache/Homebrew/downloads/9e3339003cb5ba538e5cd80da6c49ffec055f92051e87e4116a7a61bdbf724ee--mpfr--4.1.0.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/libmpc/manifests/1.2.1
Already downloaded: /root/.cache/Homebrew/downloads/e62692fa3502149022fa3ae477cc4feba8b40abb6a63f9c5f96b45046749960a--libmpc-1.2.1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/libmpc/blobs/sha256:c12f0406fb375c08d4c9f699ec2d28099088bb999fe9cceefe9145c001f24f89
Already downloaded: /root/.cache/Homebrew/downloads/dc0d861d95984c0648c98041da12e9e602d1a71869ca5462ab5396fbf38c9009--libmpc--1.2.1.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/zstd/manifests/1.5.0
Already downloaded: /root/.cache/Homebrew/downloads/b00e490860c942363d52f56167baf68a471d55b7264193bf89ea10944737e7d1--zstd-1.5.0.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/zstd/blobs/sha256:35f3d0c3e35229b4b9d32a2315f061651ee1cf88e2d5216ec259f2aece4dafb3
Already downloaded: /root/.cache/Homebrew/downloads/4ff6ec35ade9236083583cb5152b48159c876226037db6e5cf28eb54e6b34a47--zstd--1.5.0.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/binutils/manifests/2.36.1
Already downloaded: /root/.cache/Homebrew/downloads/98035dca1bd69136143e969fee292c023e934986ac40cd2e17b8e85bbd55b4f8--binutils-2.36.1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/binutils/blobs/sha256:d13d509bb08d3221982d69c2f2a766c535c858de379dd0e10d4d6f2309d6f597
Already downloaded: /root/.cache/Homebrew/downloads/ca1e5a66e06527a6c68aa2eb3bb90139713432a7793d996a450986542e02a992--binutils--2.36.1.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/gcc/manifests/11.1.0_1
Already downloaded: /root/.cache/Homebrew/downloads/ad7bfe11c97fd11b926e1ab7ebba83223ea95394feaa7b4ab5082612a08eec3b--gcc-11.1.0_1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/gcc/blobs/sha256:7a2f47441a55ad77374ad67674d89fca5def4f9585ada08352d6498a1771f361
Already downloaded: /root/.cache/Homebrew/downloads/c22097ffef17ea1fb4dab8bd71a8eb0eeb0977c7fb7b0da4b3dfc2bac4b3f734--gcc--11.1.0_1.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/pkg-config/manifests/0.29.2_4
Already downloaded: /root/.cache/Homebrew/downloads/6698c640583800d8bd7210efd9067cb6b647b314b385e71eaae24b8b3e96a3f9--pkg-config-0.29.2_4.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/pkg-config/blobs/sha256:bc8ac04f3d8e42a748f40544f8e1b7f2471f32608f33e666e903d6108eb4dab2
Already downloaded: /root/.cache/Homebrew/downloads/e272fd0a126a2a6faf34efa96fc5ce45becceac07323a715616697dfb3a280c1--pkg-config--0.29.2_4.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/libpthread-stubs/manifests/0.4
Already downloaded: /root/.cache/Homebrew/downloads/4aa4164e3fadc3d825507c50bf28fcf5ada36fd0365b56db6d648acd2985cb5a--libpthread-stubs-0.4.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/libpthread-stubs/blobs/sha256:56f9851df3cdf76b43e63cab025c742ff54c62794c5a0bc93cf4364323d77421
Already downloaded: /root/.cache/Homebrew/downloads/0483b2521e8baa5e143445c071063345cd3e951b7f4c144c2b6e83f2175a8a89--libpthread-stubs--0.4.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/xorgproto/manifests/2021.4
Already downloaded: /root/.cache/Homebrew/downloads/d4a813c6651a7dceb4f0a0d61303e1fb72d572dec4e1d107c879264160f8fa2e--xorgproto-2021.4.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/xorgproto/blobs/sha256:f8a7c3b5c61c48affa144576747aa93a18c49f8b861b782e45aeb643a2708363
Already downloaded: /root/.cache/Homebrew/downloads/3ed0295befd21880f05c88d338780945dc27fb504a027bf11d98051fcabe05d1--xorgproto--2021.4.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/libxau/manifests/1.0.9
Already downloaded: /root/.cache/Homebrew/downloads/cd9f3e81bca5715f649901e68881d9a35f02617fb11abad5555c096b729bad1b--libxau-1.0.9.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/libxau/blobs/sha256:6be192d4f2b68b34f86d0a1a2e8e5324a2124bca531972464942e56cc4339e8d
Already downloaded: /root/.cache/Homebrew/downloads/9f8f20f38a4ca35305d1d8df001fbe5c4cda2c13e4a0bf4a477611b03126872f--libxau--1.0.9.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/libxdmcp/manifests/1.1.3
Already downloaded: /root/.cache/Homebrew/downloads/8841483d656bb0f9bdfcd9f1099ee878bcf022242e58da979a3b6ad149aa3343--libxdmcp-1.1.3.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/libxdmcp/blobs/sha256:2f3a17fca2a8c87f3b6b6c7b9dc170b90b64d1fe8ad46b4f9b63fe2513bacc25
Already downloaded: /root/.cache/Homebrew/downloads/77e0f7a400d2cac626cb69a54e38be9e58020933d7d999daf7de8e46514e12ab--libxdmcp--1.1.3.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/libxcb/manifests/1.14_1-1
Already downloaded: /root/.cache/Homebrew/downloads/f5872e2e1d16755472bece2fb1abe186104d630a9b0aad6a0e58172775304bf6--libxcb-1.14_1-1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/libxcb/blobs/sha256:1a4eed8760ef7dc1a20d5a5b0f4da7430738a08c2bb7dddc97766a6bf4ee2ff5
Already downloaded: /root/.cache/Homebrew/downloads/0c3121e4b48042cd82074be76dbf9a1a8225b51939f8a3192ed58c20fb145118--libxcb--1.14_1.x86_64_linux.bottle.1.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/libx11/manifests/1.7.2
Already downloaded: /root/.cache/Homebrew/downloads/ffcaac7ad3b03d59d72575b92a1a580aa50873dbf669eee4e2d2d280774148c3--libx11-1.7.2.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/libx11/blobs/sha256:035f49996ae8031ba4531113482e789feb51abeb8877eb8c0681ea615ff8cd93
Already downloaded: /root/.cache/Homebrew/downloads/9adaea4752ce78c0e0776325f0234c44d2f13fb678d6e318c6c3b6573b50127c--libx11--1.7.2.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/libxext/manifests/1.3.4
Already downloaded: /root/.cache/Homebrew/downloads/ebc0a7db9caa1e6c721c918ae5ff09b468f21b978de1992daa8ffcee416f9555--libxext-1.3.4.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/libxext/blobs/sha256:ec21c900415cb14a9e86a96b00d9c2eaa1c22b4e79babad8748ac072ba52252b
Already downloaded: /root/.cache/Homebrew/downloads/59641f3a773d5ff70218d62790c707c90182cf8e4eeaff6aa30b8f05bf8c3f54--libxext--1.3.4.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/libxrender/manifests/0.9.10
Already downloaded: /root/.cache/Homebrew/downloads/20bb32f852fe98627c26f093e27fc2a46e3f7183ad43f371a701a4a75d48ddfc--libxrender-0.9.10.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/libxrender/blobs/sha256:f00b409bc70ad54a54f07f3605a6c0be3fdc196ac330e9376a6183d92ad495c0
Already downloaded: /root/.cache/Homebrew/downloads/6a4281b651de247823d621407cdde736db06fc8a1e1a33c4ec77e123cb656339--libxrender--0.9.10.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/libxrandr/manifests/1.5.2
Already downloaded: /root/.cache/Homebrew/downloads/3afa381d4320870b7870f579c4ae4b6fd9eae47c35f1603ef829aed3a0d1faf1--libxrandr-1.5.2.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/libxrandr/blobs/sha256:dde0fd3c63a0471016e822a59d83826b530104047c0962163fff97ff06d84eb2
Already downloaded: /root/.cache/Homebrew/downloads/70c650d070f9d3a8187a92a60f282e97d13201fc2751df594e830ae0e1c2f1f6--libxrandr--1.5.2.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/libice/manifests/1.0.10
Already downloaded: /root/.cache/Homebrew/downloads/a6a5e7f8c757de02eb9c6db47686fbe86cef84a4954600261436aa4fda073c19--libice-1.0.10.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/libice/blobs/sha256:935f190724128149ac5c9d38202080532a071a4e8ae1248a794824a9dee61f12
Already downloaded: /root/.cache/Homebrew/downloads/3452ff350bfdde4b90d551df9cc188dd2c340c8f2816fc855782c75ba330377d--libice--1.0.10.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/libsm/manifests/1.2.3
Already downloaded: /root/.cache/Homebrew/downloads/0f3e7673c8e2d03d985a0e809f06f08b671f720bc864fa06af84574b03961a3f--libsm-1.2.3.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/libsm/blobs/sha256:a950f4644e491d2fcadb37db2e96554d5bae2f5446dce62c57cf7398b89645d1
Already downloaded: /root/.cache/Homebrew/downloads/8f470e534fb805e3700baacfc3871fe9015464c85e5792620671401cbcc4db70--libsm--1.2.3.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/libxt/manifests/1.2.1
Already downloaded: /root/.cache/Homebrew/downloads/f9476b119123f31de221ec8397905b68ffa4803ef77ce52136060367ead8ae9b--libxt-1.2.1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/libxt/blobs/sha256:80cf4172680aeb30f00d79f4058df28915598ecc9cfed5e63e447b9b63507dbb
Already downloaded: /root/.cache/Homebrew/downloads/7ac1be5d0e31fcbcfd2d5f37162d6e4364f799962bb8ee2f8d6a766111471c84--libxt--1.2.1.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/libxfixes/manifests/6.0.0
Already downloaded: /root/.cache/Homebrew/downloads/4a5dcbee895967570b59874815b725c74098c5fe1cef19c658f4cf8c912cfc1d--libxfixes-6.0.0.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/libxfixes/blobs/sha256:b10945be16e84b8e7cd08c600d64408e1332574f1fa8e33a44aadb3bf5e06d68
Already downloaded: /root/.cache/Homebrew/downloads/cccb975a9af424863be745b82175e46b836578f0f1c43524a5bc189016ead670--libxfixes--6.0.0.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/libxi/manifests/1.7.10
Already downloaded: /root/.cache/Homebrew/downloads/9add7e2024d29f66e295329381dabadce368a62900e63b43697097ee69f2f2c9--libxi-1.7.10.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/libxi/blobs/sha256:3f8973202e820185cf4976d01580f63ef9f43d299f7a983415abc08a5e658a1d
Already downloaded: /root/.cache/Homebrew/downloads/c8019268a5a072c58af38c9604f46d9b5251390459df7807038d0d5a936e3415--libxi--1.7.10.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/libxtst/manifests/1.2.3
Already downloaded: /root/.cache/Homebrew/downloads/4e75b8f815657703816ba62c75683713eff3e169959d56b2bbb673496d2ba2f6--libxtst-1.2.3.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/libxtst/blobs/sha256:683834d974c399696213f15dffff6104240d75436def1c233f4f490198913bdd
Already downloaded: /root/.cache/Homebrew/downloads/819fd1ed9ba324e848a9e79a5dad16eb63144c99d7ca0cb9919cc1b1d215c13d--libxtst--1.2.3.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/unzip/manifests/6.0_7
Already downloaded: /root/.cache/Homebrew/downloads/8ea7160a83a820cc43d63df5947fadf42bc7cfa8dd436fc45a552b641eea0b3e--unzip-6.0_7.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/unzip/blobs/sha256:7f64b1d75f83c936125e0ae873668c992ca1f2041480f899b178959da4cafa1b
Already downloaded: /root/.cache/Homebrew/downloads/980715cb7e99d8549c7b8e3454ee0ab681dab5bb343a0bec8390c77700368837--unzip--6.0_7.x86_64_linux.bottle.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/zip/manifests/3.0-1
Already downloaded: /root/.cache/Homebrew/downloads/3f174f8ff2bba0f8cd8540d8ad6ffd85a47ee03a4b74f07537a104d13a536806--zip-3.0-1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/zip/blobs/sha256:eee82c511bc78167c6bc043a8795f3b7b47e2ba77a595d6234648ae73102efb2
Already downloaded: /root/.cache/Homebrew/downloads/453d4d73a17809089c294cd7160f63e172abb6a579304304f0da1d09e0d492ce--zip--3.0.x86_64_linux.bottle.1.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/openjdk/manifests/16.0.1-1
Already downloaded: /root/.cache/Homebrew/downloads/9656b3565abe6b3bb2cd9284a32c50aa2d9d7c6ad12922b6737e5fbb540f4248--openjdk-16.0.1-1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/openjdk/blobs/sha256:3f0605079feaec84c5b944c16975cc6283bb4f7074bae283ac59d45492fa5a81
Already downloaded: /root/.cache/Homebrew/downloads/3e78590912a3353959e408fe603bc1ed83388597ac194324d6dabb5ab939de90--openjdk--16.0.1.x86_64_linux.bottle.1.tar.gz
==> Downloading https://ghcr.io/v2/linuxbrew/core/flyway/manifests/7.10.0
curl: (22) The requested URL returned error: 404                              

Error: Failed to download resource "flyway_bottle_manifest"
Download failed: https://ghcr.io/v2/linuxbrew/core/flyway/manifests/7.10.0
```

### What did you expect to happen?

Install flyway

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
`# brew install flyway`
```

**Comment by carlocab** [comment]
CC @Homebrew/linux

**Comment by iMichka** [comment]
Fixed by 90f03b99bac3614cb87c174c1616beec0e820469.

This was an issue with an `all:` bottle that was not cleaned up.
